trip_journal_app/migrations/0004_confirmation_code.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
is there any way to avoid importing from future? Anyway, I don't see it ever used.